TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -701,7 +697,6 @@ void menuPrinting(void)
         }
         else
         { // Main button
-          clearInfoFile();
```

Comment:
Do not leave garbage in memory. From this point on  the data is not needed anymore, it is just a waste of precious memory space. Yes, it will be cleared in the next print but why keep it in the memory???
Set it free! :)

Reply:
it was already cleared at the beginning of this print (not the next one) in initMenuPrinting().

Reply:
Yes, you're right. In this case it should be removed from "menu.c" (line 1232~1233) also.